plugins/outputs/file/file.go

```diff
@@ -21,8 +23,10 @@ type File struct {
 
 var sampleConfig = `
   ## Files to write to, "stdout" is a specially handled file.
+  ## Will also accept fromats such as /tmp/{020106}/metrics{020106.150406}.out
```

Comment:
Hi, are you waiting on me to do something on this ? - I believe I have covered everything requested, but if not, please let me know.